pynamodb/models.py

```diff
@@ -1177,6 +1182,17 @@ def _get_indexes(cls):
                     cls._indexes[pythonic(LOCAL_SECONDARY_INDEXES)].append(idx)
         return cls._indexes
 
+    def _set_ttl_field(self):
+        """
+        Sets the TTL field specified in the Meta if the field is not set.
+        """
+        cls = type(self)
+        if cls.Meta.ttl_field and cls.Meta.ttl_seconds:
```

Comment:
Should we error preemptively if the TTL attribute's type is not `N`?

Reply:
Ah yes.. I was thinking about that.  I wanted to make sure the approach was ok